lib/SPIRV/libSPIRV/SPIRVEntry.cpp

```diff
@@ -388,11 +388,11 @@ SPIRVEntry::setLinkageType(SPIRVLinkageTypeKind LT) {
   addDecorate(new SPIRVDecorateLinkageAttr(this, Name, LT));
 }
 
-std::ostream &
-operator<<(std::ostream &O, const SPIRVEntry &E) {
+llvm::raw_ostream &
+operator<<(llvm::raw_ostream &O, const SPIRVEntry &E) {
   E.validate();
   E.encodeAll(O);
-  O << SPIRVNL;
+  SPIRVNL(O);

```

Comment:
A local var is fine. Since the code is so simple, compiler should be able to eliminate the local var.